pkg/authorization/authorizer/scope/converter.go

```diff
@@ -135,6 +135,16 @@ const (
 	UserFull = UserIndicator + "full"
 )
 
+func DefaultSupportedScopes() []string {
+	return []string{
```

Comment:
Make this list a private var and have this function return it (convention we use everywhere).